projects/cdk/services/static-request.service.ts

```diff
@@ -23,6 +26,17 @@ export class TuiStaticRequestService {
             return cache;
         }
 
+        if (!url.startsWith(`http`)) {
+            const hostname = `${this.locationRef.protocol}//${this.locationRef.hostname}`;
+            const port = this.locationRef.port ? `:${this.locationRef.port}` : ``;
+            const baseUrl =
+                `${hostname}${port}/` !== this.baseHref
+                    ? `${hostname}${port}/${this.baseHref}`
+                    : this.baseHref;
+
+            url = `${baseUrl}/${url}`.replace(/(https?:\/\/)|(\/)+/g, `$1$2`);
```

Comment:
I confirm that there are no more warning in console.

But icons still do not work during SSR 🤔 
<img width="1787" alt="Screenshot 2023-10-18 at 13 22 07" src="https://github.com/taiga-family/taiga-ui/assets/35179038/c0d01efc-f58e-403e-980b-4e90107b2311">

Reply:
@splincode 
Should we merge this PR and create the separate issue for absence of icons ?

Or it can be solved in this PR?